pipelines/ocp-4.17-pipeline.yml

```diff
@@ -8,13 +8,12 @@
   dci_credentials: ~/.config/dci-pipeline/dci_credentials.yml
   ansible_extravars:
     dci_cache_dir: ~/dci-cache-dir
-    dci_config_dirs:
-      - ~/config/ocp-install
+    dci_config_dir: ~/config/ocp-install
```

Comment:
No need for a change here, that was correct. for the ocp agent, it is `dci_config_dirs` with a list.

Reply:
oh I see, let me fix the KS PR as well.
Would you prefer to close this PR or just keep 4.17 changes?